packages/styles/src/link/index.css

```diff
@@ -0,0 +1,82 @@
+/**
+ * Link styles
+ * @variant {small | medium | large} syn-link The size of the link
+ * @boolean { false } syn-link--quiet Applies the quiet link styling
+ * @boolean { false } syn-link--disabled Applies the disabled link styling
+ */
+.syn-link {
+  align-items: center;
+  color: var(--syn-link-color);
+  display: inline-flex;
+  font: var(--syn-body-medium-regular);
+  gap: var(--syn-spacing-x-small);
+}
+
+/* Use inline text styling when there is no syn-icon */
+.syn-link:not(:has(syn-icon)) {
+  display: inline;
+}
+
+.syn-link:not(.syn-link--disabled):hover {
+  color: var(--syn-link-color-hover);
+}
+
+.syn-link:not(.syn-link--disabled):active {
+  color: var(--syn-link-color-active);
+}
+
+.syn-link:not(.syn-link--disabled):focus-visible {
+  outline: var(--syn-focus-ring);
+  outline-offset: var(--syn-focus-ring-offset);
+}
+
+/* Overrides for smaller links */
+.syn-link--small {
+  font: var(--syn-body-small-regular);
+  gap: var(--syn-spacing-2x-small);
+}
```

Comment:
The icons are way bigger in design.

```
.syn-link--small syn-icon {
    font: var(--syn-spacing-medium);
}
```
Maybe this would fit. Check out the sizes in Figma.

Reply:
Yes, there where no docs yesterday, adjusted this now